apis/io-engine/src/v0.rs

```diff
@@ -36,7 +36,7 @@ include!(concat!(env!("OUT_DIR"), "/mayastor.rs"));
 // This has to be done every time Mayastor API V0 is changed.
 #[cfg(not(target_os = "linux"))]
 mod v0_generated {
-    include!("../../../.pregenerated/mayastor.rs");
+    include!(".pregenerated/mayastor.rs");
```

Comment:
Is this really necessary? Doesn't mac generate this too?
I remember something like this was causing an issue before on the macos builds on github actions.